### ISSUE TITLE
[Accessibility]A11y__AzureDataStudio_CSVModifyColumns _Keyboard:Table Dropdown list is not accessible through keyboard.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#6070]
[Check out Keros! ](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx)- Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yMAS; #A11yTCS; #MAS2.1.1; #A11y_AzureDataStudio; #Win32; #DesktopApp

**Environment Details:**
Application name: Azure Data Studio
Version: 1.9.0-insider (user setup)
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS:MacOS Mojave 10.14.5
**Additional Details:**
Tool Name: Keyboard
MAS Reference:MAS2.1.1,
**Repro Steps:**
1.Launch the Azure Data Studio application.
2.Navigate to the Data services folder 
3.Go to HDFS Folder
4.Navigate to CSV files folder
5.Right click on the files->Select option Create external table from CSV
6.Click on the "Next" button->Click on next button.
**Actual:**
Issue 1.when we Navigate through the CSVModifyColumns table drop down through keyboard and hit space bar to open it and try to navigate inside the drop down list but we are not able to navigate inside the drop down list by keyboard.

**Expected:**
Drop down list should be accessible through keyboard.
User Impact:
User with motor disabilities will find it difficult to navigate through the drop down.
**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx
**MAS Reference**:
**MAS2.1.1,MAS1.3.1**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
**Attachment for Reference:**
[6070_[Accessibility]A11y__AzureDataStudio_CSVModifyColumns _Keyboard.pptx](https://github.com/microsoft/azuredatastudio/files/3300837/6070_.Accessibility.A11y__AzureDataStudio_CSVModifyColumns._Keyboard.pptx)
Does this issue occur when all extensions are disabled?: Yes

**Comment by Charles-Gagnon** [comment]
This appears to be fixed in the latest build on master
